HintList.py

```diff
@@ -198,79 +221,763 @@ def getRequiredHints(world):
+    'Fire Temple MQ Map Room Side Chest':                          ("a #falling slug# in the Fire Temple guards", None, 'exclude'),
+    'Fire Temple MQ Map Chest':                                    ("using a #hammer in the depths of the Fire Temple# reveals", None, 'exclude'),
+    'Fire Temple MQ Boss Key Chest':                               ("#illuminating a lava pit# reveals the path to", None, 'exclude'),
+    'Fire Temple MQ Big Lava Room Blocked Door Chest':             ("#explosives over a lava pit# unveil", None, 'exclude'),
+    'Fire Temple MQ Lizalfos Maze Side Room Chest':                ("a #Goron hidden near a maze# holds", None, 'exclude'),
+    'Fire Temple MQ Freestanding Key':                             ("hidden #beneath a block of stone# lies", None, 'exclude'),
+
+    'Water Temple Map Chest':                                      ("#rolling spikes# in the Water Temple surround", None, 'exclude'),
+    'Water Temple Compass Chest':                                  ("#roaming stingers in the Water Temple# guard", None, 'exclude'),
+    'Water Temple Torches Chest':                                  ("#fire in the Water Temple# reveals", None, 'exclude'),
+    'Water Temple Dragon Chest':                                   ("a #serpent's prize# in the Water Temple is", None, 'exclude'),
+    'Water Temple Central Bow Target Chest':                       ("#blinding an eye# in the Water Temple leads to", None, 'exclude'),
+    'Water Temple Central Pillar Chest':                           ("in the #depths of the Water Temple# lies", None, 'exclude'),
+    'Water Temple Cracked Wall Chest':                             ("#through a crack# in the Water Temple is", None, 'exclude'),
+    'Water Temple Longshot Chest':                                 (["#facing yourself# reveals", "a #dark reflection# of yourself guards"], "#Dark Link# guards", 'exclude'),
+
+    'Water Temple MQ Central Pillar Chest':                        ("in the #depths of the Water Temple# lies", None, 'exclude'),
+    'Water Temple MQ Boss Key Chest':                              ("fire in the Water Temple unlocks a #vast gate# revealing a chest with", None, 'exclude'),
+    'Water Temple MQ Longshot Chest':                              ("#through a crack# in the Water Temple is", None, 'exclude'),
+    'Water Temple MQ Compass Chest':                               ("#fire in the Water Temple# reveals", None, 'exclude'),
+    'Water Temple MQ Map Chest':                                   ("#sparring soldiers# in the Water Temple guard", None, 'exclude'),
+
+    'Spirit Temple Child Bridge Chest':                            ("a child conquers a #skull in green fire# in the Spirit Temple to reach", None, 'exclude'),
+    'Spirit Temple Child Early Torches Chest':                     ("a child can find a #caged chest in Spirit Temple# with", None, 'exclude'),
```

Comment:
"in the Spirit Temple"?

Reply:
moved text outside of the hashes to be consistent with other hints

Reply:
I meant there should be a "the" to be consistent with other hints.

Reply:
Whoops! My brain skipped right over that. Fixed now.